packages/material-ui-codemod/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "material-ui-codemod",
+  "version": "0.1.0",
+  "description": "Codemod scripts for Material-UI",
+  "scripts": {},
+  "license": "MIT"
+}
```

Comment:
I think npm will warn about missing keys when you publish. `repository` and `license` are mandatory I think, and possibly `author`. Some material-ui related `keywords` might be a good idea to help discoverability.